src/components/InfinityModal/InfinityModalForm.tsx

```diff
@@ -0,0 +1,330 @@
+	const tnLabel = <span>TN <InfoTooltip placement="bottom" content={tooltip.tnInfo} /></span>;
+	const untrainedTestLabel = <span>Untrained Test <InfoTooltip placement="bottom" content={tooltip.untrainedTestInfo} /></span>;
+
+	const assistanceFocusLabel = <span>Foc <InfoTooltip placement="bottom" content={tooltip.assistanceFocusInfo} /></span>;
+	const assistanceTnLabel = <span>TN <InfoTooltip placement="bottom" content={tooltip.assistanceFocusInfo} /></span>;
+
+	return (
+		<Form
+			className={(invalid && (submitFailed || anyTouched)) ? 'form-invalid' : ''}
+			id="infinity-mode-form"
+			onSubmit={handleSubmit}>
+			<div className="skill-level-field infinity-skill-level-field">
+				<div className="infinity-field">
+					<Field
+						id="focus"
+						name="focus"
+						label={focusLabel}
+						onChange={(evt => handleFocusChange(evt.currentTarget.value))}
+						component={renderInput}
+					/>
+
+				</div>
+				<div className="infinity-field">
+					<Field
+						id="tn"
+						name="tn"
+						label={tnLabel}
+						component={renderInput}
+					/>
+
+				</div>
+			</div>
+			<div className="skill-level-field infinity-skill-level-field">
+				<Field
+					id="difficulty"
+					name="difficulty"
+					component={DiffLadder}
+				/>
+			</div>
+			<div className="flex-center">
+				<div className="infinity-radio-fields infinity-radio-fields--dice-to-roll">
+					<InfinityDiceRow
+						dice={dice}
+						diceMax={5}
+						fortune={fortune}
+						hoverState={hoverState}
+						handleOnHover={setHoverState}
+						handleOnClick={handleDiceChange}
+						isAssistance={false}
+					/>
+				</div>
+			</div>
+			<Field
+				name="untrainedTest"
+				id="untrainedTest"
+				label={untrainedTestLabel}
+				component={RenderCheckbox}
+				disabled={focus && Number(focus) > 0}
+			/>
+			<div className="fortune">
+				<h5 className="fortune-title">Fortune <InfoTooltip content={tooltip.fortuneInfo} /></h5>
+				<div className="infinity-radio-fields">
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="0"
+						/>
+							None
+						</label>
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="1"
+							onChange={() => Number(dice) < 3 ? change('dice', '3') : null}
+						/>
+							One
+						</label>
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="2"
+							onChange={() => Number(dice) < 4 ? change('dice', '4') : null}
+						/>
+							Two
+						</label>
+					<label className="dice-row-label">
+						<Field
+							name="fortune"
+							component="input"
+							type="radio"
+							value="3"
+							onChange={() => Number(dice) < 5 ? change('dice', '5') : null}
+						/>
+							Three
+						</label>
+				</div>
+			</div>
+			<div className="assistance">
```

Comment:
Whole "assistance" block seem to be copy-pasted from Conan modal. Could it be extracted as a module?

Reply:
I think so, yes.   I would earmark that for later update unless you need it as module before merge.